accelerator/server.cc

```diff
@@ -110,6 +110,32 @@ int main(int, char const**) {
         res << json_result;
       });
 
+  /**
+   * @method {post} /transaction send transfer
+   *
+   * @return {String} transaction object
+   */
+  mux.handle("/transaction")
+      .post([&](served::response& res, const served::request& req) {
+        char* json_result;
+
+        if (req.header("content-type") != "application/json") {
```

Comment:
Does operator `!=` perform byte-by-byte character comparison?

Reply:
According to spec **`21.4.8.3 operator !=`**
The above comparison fit this situation:
```
template<class charT, class traits, class Allocator>
   bool operator!=(const basic_string<charT,traits,Allocator>& lhs,
                            const charT* rhs);
Requires: rhs points to an array of at least traits::length(rhs) + 1elements of charT.
Returns: lhs.compare(rhs) != 0
```
And `compare` performs byte-by-byte character comparison, in [1] it uses the phrase **_lexicographical comparison_**([2]).

references:
* 1: [char_traits::compare](http://www.cplusplus.com/reference/string/char_traits/compare/)
* 2: [lexicographical_compare](http://www.cplusplus.com/reference/algorithm/lexicographical_compare/)